tamil/index.php

```diff
@@ -10,7 +10,7 @@
 ?>
 <h2 class="red underline large">Keyman for Tamil99</h2>
 <p>
-    Type in Tamil on iPhone, Windows and Android. Our Tamil keyboards works with Microsoft Word, Photoshop, Facebook, Twitter, email and thousands of other applications. Free trial available!
+    Type in Tamil on iPhone, Windows and Android. Our Tamil keyboards works with Microsoft Word, Photoshop, Facebook, Twitter, email and thousands of other applications.
 </p>
```

Comment:
You seem to have missed the download box on this page?

Reply:
No, the keyboard sources don't claim that they support Linux

Reply:
It should support Linux -- these keyboards were just released before Keyman for Linux was a thing and specified windows and macos instead of desktop, I think. We should create an issue to update the metadata and test [on the keyboards site](https://github.com/keymanapp/keyboards/issues/new).

Reply:
https://github.com/keymanapp/keyboards/issues/1235